Helper/Sale.php

```diff
@@ -74,12 +74,20 @@ function getSales()
 
             $productID = $item->getData('product_id');
             $quantity = (string)($item->getData('qty_ordered') - ($item->getData('qty_canceled') + $item->getData('qty_refunded')));
-
+            
+            // This has returned "" in the wild
             $storeId = $item->getData('store_id');
-            $zones = $this->getTimezone(array($storeId));
-            $zone = $zones[$storeId];
-            $dt = new \DateTime($item->getData('created_at'), new \DateTimeZone($zone));
-            $createdAt = $dt->format('Y-m-d H:i:sP');
+
+            $createdAt = null;
+            if(!empty($storeId)){
+                $zones = $this->getTimezone();
+                $zone = $zones[$storeId];
+                $dt = new \DateTime($item->getData('created_at'), new \DateTimeZone($zone));
+                $createdAt = $dt->format('Y-m-d H:i:sP');
+            } else {
+                $dt = new \DateTime($item->getData('created_at'));
```

Comment:
what's this `\` ? seems like half a comment lol. damn PHP

Reply:
Seems to be used to refer to the global namespace -> https://stackoverflow.com/questions/4790020/what-does-a-backslash-do-in-php-5-3